spec/system/composer/default_to_subcategory_spec.rb

```diff
@@ -41,13 +41,9 @@
             end
           end
           describe "Category does not have subcategory" do
-            it "should have the 'New Topic' button enabled and default Subcategory set to latest default subcategory" do
+            it "should have the 'New Topic' button disabled" do
```

Comment:
The two updated specs in this file seemed wrong to me. The `default_subcategory_on_read_only_category` should not affect the new-topic button behaviour on categories which have no subcategories.